src/ensureArray.js

```diff
@@ -0,0 +1,23 @@
+import { when, of } from 'ramda';
+import { isNotArray } from './index';
+
+
+/**
+ * Returns an array, even if not, wrap it in a singleton one.
```

Comment:
If the argument is already an Array, return it, otherwise return a new Array with the argument as its only element.